src/components/EditorSideBar/EditorSideBarContent.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box } from "@chakra-ui/react";
+import React from "react";
+import { ChevronLeftIcon } from "@chakra-ui/icons";
+
+interface Props {
+  iconClick: string;
+  onHandleCloseClick: () => void;
+}
+const SideBarContainer = (props: Props) => {
+  return (
+    <Box
+      bg="background.secondary"
+      w="356px"
+      h="calc(100vh - 72px)"
+      top="72px"
+      left="96px"
+      position="fixed"
+      z-index="10"
+      color="#fff"
+      p={6}
+    >
+      {props.iconClick}
```

Comment:
This iconClick on here may not be a proper name for this variable. It is the components this side menu would render on the screen. Try to use some meaningful variable name for it.

Reply:
Got it. Update with "iconClickTitle".